StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/IntentConfirmParams.swift

```diff
@@ -108,6 +108,7 @@ class IntentConfirmParams {
         }
     }
     func setAllowRedisplay(paymentMethodSave: Bool?,
+                           allowRedisplayOverride: STPPaymentMethodAllowRedisplay?,
```

Comment:
nit: I think this method could be called with invalid params, right? Like `setAllowRedisplay(paymentMethodSave: false, allowRedisplayOverride: .limited)`

Maybe it should take a `CustomerSession?` to eliminate that

Reply:
Good point, we can address this by passing in an optional "PaymentSheetComponentFeature".  Will address in subsequent PR when hiding checkbox.

Reply:
Addressed here:
https://github.com/stripe/stripe-ios/pull/3884